pyanaconda/modules/subscription/subscription.py

```diff
@@ -229,8 +229,13 @@ def process_kickstart(self, data):
             try:
                 proxy = util.ProxyString(data.rhsm.proxy)
                 if proxy.host:
+                    # ensure port is an integer and set to -1 if unknown
+                    port = -1
+                    if proxy.port:
+                        port = int(proxy.port)
```

Comment:
Please, use `port = int(proxy.port) if proxy.port else -1`.

Reply:
Hmm, yeah, this looks better. Changed.